services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -693,7 +759,8 @@ public void processDlqRequestRejected() {
      */
     @Test
     public void processDlqRequestForwarded() {
-        final String requestStatus = "FORWARDED";
+
+        final String requestStatus = FORWARDED;
```

Comment:
The requestStatus variable is redundant and can be removed.  The FORWARDED constant can be used in place of it.

Reply:
Line is still present.

Reply:
done